### ISSUE TITLE
Can't get it working

### ISSUE THREAD

**Issue by dedepete** [viswaramamoorthy/gradle-util-plugins#1]
Hi there! We've been trying to apply this plugin in the project FastHub, but it still throws `CreateProcess error=206`.  
We have also tried to manipulate with `apply` location without any success.  

Here are commits and build results:  

| Commit | AppVeyor CI Build |
|:---------------------------------------------------------------------------------------------:|:----------------------------------------------------------:|
| https://github.com/k0shk0sh/FastHub/pull/798/commits/5e84f79b8d6e2aff41095d359f03c2cfcc0cd12d | https://ci.appveyor.com/project/k0shk0sh/fasthub/build/342 |
| https://github.com/k0shk0sh/FastHub/pull/798/commits/c1bc5b141695fae5c8da9a860f0bab09e630521b | https://ci.appveyor.com/project/k0shk0sh/fasthub/build/345 |
| https://github.com/k0shk0sh/FastHub/pull/798/commits/bc787c26c0932889f363348c891cb2910c19d9f3 | https://ci.appveyor.com/project/k0shk0sh/fasthub/build/347 |

Can you help us, please?

**Comment by viswaramamoorthy** [comment]
ManifestClasspath is for handling long classpath (> 32 KB length) passed to JavaExec gradle task. Is your issue related to long classpath? Could you share output of gradle with --info flag?

**Comment by dedepete** [comment]
To our surprise, we can reproduce the problem no more. I will reopen the issue if it will occur again.  
Sorry for inconvenience.

**Comment by yakov116** [comment]
@viswaramamoorthy it hapened again
https://ci.appveyor.com/project/yakov116/fasthub-afi9h/build/1.0.8

**Comment by viswaramamoorthy** [comment]
Log file that you shared does show any indication of Manifest plugin being
invoked. It makes me think that the
:app:transformClassesWithDesugarForDebug task is not of type JavaExec.
Assuming that you profile file picks up Manifest plugin, I do not have any
idea about :app:transformClassesWithDesugarForDebug, I could not comment
further or identify a solution. I did some searching and I could not find
more information about :app:transformClassesWithDesugarForDebug task or its
source code. Do you know if :app:transformClassesWithDesugarForDebug task
is part of open source? If so, send me location of source for
:app:transformClassesWithDesugarForDebug. I could check some more and get
back to you

Basically I am looking for task type for
:app:transformClassesWithDesugarForDebug. With that Manifest plugin can be
expanded to apply the fix for that task type as well.

**Comment by yakov116** [comment]
@viswaramamoorthy thNks for replying

The issue happened when we forked firebase. I will link you to the files later when I am at a computer.



_Sent from my Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_
